tests/kafkatest/tests/connect/connect_distributed_test.py

```diff
@@ -418,7 +418,7 @@ def test_bounce(self, clean):
         if not success:
             self.mark_for_collect(self.cc)
             # Also collect the data in the topic to aid in debugging
-            consumer_validator = ConsoleConsumer(self.test_context, 1, self.kafka, self.source.topic, consumer_timeout_ms=1000, print_key=True)
+            consumer_validator = ConsoleConsumer(self.test_context, 1, self.kafka, self.source.topic, new_consumer=False, consumer_timeout_ms=1000, print_key=True)
```

Comment:
Not required if you don't want to, but I think it's reasonable/safe to change here. I think the only time we should use the old consumer is when it is the target under test. Here we appear to just need _some_ consumer for additional validation.